omod/src/main/java/org/openmrs/module/messages/resources/ActorResponseRestResource.java

```diff
@@ -0,0 +1,243 @@
+package org.openmrs.module.messages.resources;
```

Comment:
please use google-java-format plugin (https://github.com/google/google-java-format) to reformat code, because there are a lot of ugly indents, missing spaces etc. You can install such plugin directly in your IDE